bitcoin/script.h

```diff
@@ -153,9 +166,15 @@ bool is_p2wsh(const u8 *script, struct sha256 *addr);
 /* Is this (version 0) pay to witness pubkey hash? (extract addr if not NULL) */
 bool is_p2wpkh(const u8 *script, struct bitcoin_address *addr);
 
-/* Is this one of the four above script types? */
+/* Is this a taproot output? (extract xonly_pubkey bytes if not NULL) */
+bool is_p2tr(const u8 *script, u8 *xonly_pubkey);
```

Comment:
Nitpick: as a hint, can we declare this as `bool is_p2tr(const u8 *script, u8 xonly_pubkey[32]);` ?  It's the same, but makes it clear that it should be 32 bytes long...

Reply:
done